### DISCUSSION TITLE
How can I partial download a file with google-drive3?

### DISCUSSION THREAD

**Discussion by Yesterday17** [Byron/google-apis-rs#374] [Q&A] [answered]
Seems `Range` header is not supported?

**Comment by Byron** [comment, marked as answer]
It doesn't look like custom headers are support by the method builder, so setting the `Range` is indeed unsupported unless the API in question has documented special ways of doing that.

As a workaround, I think it should work to stop reading after a couple of bytes and drop the ondoing call, which should be streaming data - for a lack of information here I can't tell as I don't know which method is actually called.

**Reply by Yesterday17** [reply]
It seems that `Range` is documented in https://developers.google.com/drive/api/guides/manage-downloads#partial_download?
